util/metrics/src/lib.rs

```diff
@@ -79,6 +81,11 @@ static METRICS: once_cell::sync::Lazy<Metrics> = once_cell::sync::Lazy::new(|| M
         "The CKB relay transaction short id collide"
     )
     .unwrap(),
+    ckb_relay_cb_verify_time: register_histogram!(
+        "ckb_relay_cb_verify_time",
+        "The CKB relay compact block verify time"
+    )
```

Comment:
I think it's neccesery to define buckets values like:
```
    ckb_relay_cb_verify_time: register_histogram!(
        "ckb_relay_cb_verify_time",
        "The CKB relay compact block verify time",
        vec![0.0005, 0.001, 0.0015, 0.002, 0.0025, 0.003, ...]
    )
```

Reply:
the default buckets is:
```rust
/// The default [`Histogram`] buckets. The default buckets are
/// tailored to broadly measure the response time (in seconds) of a
/// network service. Most likely, however, you will be required to define
/// buckets customized to your use case.
pub const DEFAULT_BUCKETS: &[f64; 11] = &[
    0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1.0, 2.5, 5.0, 10.0,
];
```

I think it's better than an equal division buckets in this case.